elements/bulbs-slick-slideshow/bulbs-slick-slideshow.js

```diff
@@ -17,7 +19,7 @@ import './bulbs-slick-slideshow.scss';
 /// 7. On l/r keydown, trigger slidechange.
 class BulbsSlickSlideshow extends BulbsHTMLElement {
   attachedCallback () {
-    this.slideshow = $('.slider');
+    this.slideshow = $('.slider').last();
```

Comment:
yep, it really should be scoped to that. but for some reason when running this in tests this doesn't work (spent too much time digging/retrying different query selectors, no luck).